libs/api/prisma/migrations/20240313100544_add_extendable_subscription_flag_and_max_subscription_count/migration.sql

```diff
@@ -0,0 +1,11 @@
+-- AlterTable
+ALTER TABLE "member.plans" ADD COLUMN     "extendable" BOOLEAN NOT NULL DEFAULT true,
+ADD COLUMN     "maxCount" INTEGER;
+
+-- AlterTable
+ALTER TABLE "subscriptions" ADD COLUMN     "extendable" BOOLEAN NOT NULL DEFAULT true;
+
+-- Add constraint
+ALTER TABLE "subscriptions"
+    ADD CONSTRAINT "check_extendable_autorenew"
+        CHECK ("extendable" OR NOT "autoRenew");
```

Comment:
For now, it is possible to create non-extendable member-plans, with a forced auto-renew setting. We should catch this also on api level.

Reply:
![image](https://github.com/wepublish/wepublish/assets/70939054/cf4a49ab-8aa1-4a09-9395-bf712232002d)